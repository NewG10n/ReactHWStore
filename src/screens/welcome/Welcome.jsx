import React from "react";
import { Link } from "react-router-dom";

const Welcome = () => {
  return (
    <>
      <h1>Welcome to Shop Anything</h1>
      <Link to="goods">Go shopping -></Link>
    </>
  );
};

export default Welcome;
