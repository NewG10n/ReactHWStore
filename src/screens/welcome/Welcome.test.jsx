import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import Welcome from "./Welcome";

describe("Welcome component", () => {
  test("renders correctly", () => {
    const { container } = render(
      <BrowserRouter>
        <Welcome />
      </BrowserRouter>
    );
    expect(container).toMatchSnapshot();
  });
});
