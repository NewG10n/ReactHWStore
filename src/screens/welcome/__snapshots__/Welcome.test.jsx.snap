// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome component renders correctly 1`] = `
<div>
  <h1>
    Welcome to Shop Anything
  </h1>
  <a
    href="/goods"
  >
    Go shopping -&gt;
  </a>
</div>
`;
